scripts/prepare-release.mjs

```diff
@@ -0,0 +1,149 @@
+import { exec as execCallback } from "child_process";
+import fs from "fs/promises";
+import ora from "ora";
+import path from "path";
+import { fileURLToPath } from "url";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const rootDir = path.resolve(__dirname, "..");
+
+let currentChildProcess = null;
+
+class ValidationError extends Error {
+  constructor(message) {
+    super(message);
+    this.name = "ValidationError";
+  }
+}
+
+function parseArgs() {
+  const [signerAccount, signerKey, network = "mainnet"] = process.argv.slice(2);
+
+  if (!signerAccount) {
+    throw new ValidationError("Missing argument: signer account\n" +
+        "Usage: node release.js <signer account> <signer key> [network]\n\n" +
+      "Please provide the NEAR account to sign for the release.");
+  } 
+  if (!signerKey) {
+    console.warn(
+      "Missing argument: signer key\n" +
+        "Usage: node release.js <signer account> <signer key> [network]\n\n" +
+        "Will attempt to sign from keychain..."
+    );
+  }
+
+  if (network !== "mainnet" && network !== "testnet") {
```

Comment:
there is no NEARFS on testnet as far as I know

Reply:
Verified that testnet exists with a successful upload: https://ipfs.web4.testnet.page/ipfs/bafybeibe63hqugbqr4writdxgezgl5swgujay6t5uptw2px7q63r7crk2q/